config/.cmdrc_sample.json

```diff
@@ -3,7 +3,8 @@
     "REACT_APP_GQL_URL": "http://localhost:9090/graphql/query",
     "REACT_APP_API_URL": "http://localhost:3000/api",
     "REACT_APP_UI_URL": "http://localhost:9090",
-    "REACT_APP_SPRUCE_URL": "http://localhost:3000"
+    "REACT_APP_SPRUCE_URL": "http://localhost:3000",
+    "REACT_APP_GQL_COOKIE": "cookie-from-REACT_APP_GQL_URL"
```

Comment:
This should be the cookie thats used in requests when communicating with the graphql playground. Im not sure if this is different for everyone in local development and how often it changes if it does.